code/game/objects/items/devices/last_shelter.dm

```diff
@@ -20,8 +18,12 @@
 		var/obj/item/weapon/implant/core_implant/cruciform/cruciform = get_cruciform()
 		if(cruciform)
 			scan = FALSE
-			user.put_in_hands(cruciform)
-			to_chat(user, SPAN_NOTICE("The [src] has founded a losted cruciform in a deep space. Now this fate of the disciple rests in your hands."))
+			if(istype(src.loc, /mob/living/carbon/human))
+				user.put_in_hands(cruciform)
+				to_chat(user, SPAN_NOTICE("The [src] has founded a losted cruciform in a deep space. Now this fate of the disciple rests in your hands."))
```

Comment:
Spell correction : has found the lost cruciform

Reply:
Fixed